lib/OpenQA/Shared/Controller/Running.pm

```diff
@@ -112,9 +112,8 @@ sub streamtext ($self, $file_name, $start_hook = undef, $close_hook = undef) {
     # Check for new lines from the logfile using recurring timer
     # Setup utility function to close the connection if something goes wrong
     my $timer_id;
-    my $close_connection = sub ($self) {
+    my $close_connection = sub {
```

Comment:
Please do not remove the signature.

Reply:
I thought we agreed to revert this 8e91645.